src/elements/ActionMenuItem/ActionsMenuItem.js

```diff
@@ -42,6 +43,9 @@ const ActionsMenuItem = ({ dataItem, isIconDisplayed, menuItem }) => {
       key={menuItem.label}
     >
       <div
+        data-testid={`actions-menu__option-${menuItem.label
```

Comment:
We should avoid using labels in the code for purposes other than displaying them in the view.